src/battle_script_commands.c

```diff
@@ -7138,7 +7138,7 @@ u32 IsLeafGuardProtected(u32 battler)
 bool32 IsShieldsDownProtected(u32 battler)
 {
     return (gBattleMons[battler].ability == ABILITY_SHIELDS_DOWN
-            && gBattleMons[battler].species == SPECIES_MINIOR);
+            && GetFormIdFromFormSpeciesId(gBattleMons[battler].species) < 6); // Minior is not in core form
```

Comment:
I'm assuming the 6 corresponds to an entry in `sMiniorFormSpeciesIdTable` in pokemon_expansion? In this case it should be `< 7, as `SPECIES_MINIOR_CORE_RED = 7`. I also suggest:
```
#ifdef POKEMON_EXPANSION
GetFormIdFromFormSpeciesId(gBattleMons[battler].species) < SPECIES_MINIOR_CORE_RED); // Minior is not in core form
#else
GetFormIdFromFormSpeciesId(gBattleMons[battler].species) < 7);
#endif
```
But this is probably unecessary

Reply:
My two cents are that we should use the second function call, to make it easier for people to add new Minior forms should they choose to.

Reply:
Ready for re-review!